mysql-test/suite/innodb/t/general_ts_encrypt.test

```diff
@@ -0,0 +1,266 @@
+# InnoDB general tablespace encryption
+
+--source include/have_innodb.inc
+--source include/not_embedded.inc
+--source include/have_partition.inc
+
+call mtr.add_suppression("Partition.*does not exist in the InnoDB internal data dictionary though");
+
+let $innodb_file_per_table = `SELECT @@innodb_file_per_table`;
+let $innodb_strict_mode = `SELECT @@innodb_strict_mode`;
```

Comment:
Should be redundant, see the comment at the end of testcase

Reply:
Yes. Indeed. Removed this line and line 243 and re-recorded test result.